automated install/basic-install.sh

```diff
@@ -2528,6 +2528,14 @@ main() {
             # If no setting was found, default to 0
             PRIVACY_LEVEL="${PRIVACY_LEVEL:-0}"
         fi
+
+        # If this is an update from a previous Pi-hole installation
+        # we need to move any existing `pihole*` logs from `/var/log` to `/var/log/pihole`
+        # if /var/log/pihole.log is not a symlink (set durign FTL startup) move the files
+        # can be removed with Pi-hole v6.0
+        if [ -f /var/log/pihole.log ] && [ ! -L /var/log/pihole.log ]; then
+            mv /var/log/pihole*.* /var/log/pihole/ 2>/dev/null
```

Comment:
Since Pi-hole is running at this stage (?) is it probably immediately recreating the log files?

Reply:
FTL is still running at this point, but it should not recreate the file (because it only creates them on `start/restart`). All code inside `FTL` uses the new location already (at least on my current system), but I should probably move this code block further down to run it after the new FTL installation took place.
However, even if I manually remove `/var/log/pihole.log` and run `pihole restartdns` the empty file is recreated.

Reply:
Over night a few more empty files appeared

```
rockpi@rockpi-4b:~$ ls -lha /var/log/pihole*
-rw-r--r-- 1 root   root      0 May 18 00:00 /var/log/pihole_debug.log
-rw-r--r-- 1 pihole pihole    0 May 18 00:00 /var/log/pihole-FTL.log
-rw-r--r-- 1 pihole pihole    0 May 18 00:00 /var/log/pihole.log
-rw-r--r-- 1 root   root      0 May 18 00:00 /var/log/pihole_updateGravity.lo
```g

Reply:
I went through the whole install script and FTL is running at this point. It is stopped by `FTLinstall()`

https://github.com/pi-hole/pi-hole/blob/2735481da8fac3cb521338b7af2259da7b621e88/automated%20install/basic-install.sh#L2154

`FTLinstall()` is called by `FTLdetect()` which is only called once

https://github.com/pi-hole/pi-hole/blob/2735481da8fac3cb521338b7af2259da7b621e88/automated%20install/basic-install.sh#L2568

I think a good place for the files being moved is here
https://github.com/pi-hole/pi-hole/blob/2735481da8fac3cb521338b7af2259da7b621e88/automated%20install/basic-install.sh#L2615-L2616

I'm going to add a `stop_service pihole-FTL &> /dev/null` right before the restart to make sure FTL really is stopped, even if `FTLcheckUpdate` is not true.
https://github.com/pi-hole/pi-hole/blob/2735481da8fac3cb521338b7af2259da7b621e88/automated%20install/basic-install.sh#L2387